scheduling/README.md

```diff
@@ -0,0 +1,5 @@
+# Scheduling Algorithms
+
+### Python
+
+1. [Interval Scheduling](https://github.com/atin/DSA/blob/main/scheduling/python/interval_scheduling.py)
```

Comment:
according to [contribution guidelines](https://github.com/MakeContributions/DSA#contribution-guidelines) : -

- prefer relative links over absolute links.
- change the name to contain only dash(`-`) as a special character.

Reply:
Fixed it!